### ISSUE TITLE
Make it possible to freeze the header and the footer of the table

### ISSUE THREAD

**Issue by jenniferp96** [rstudio/gt#522]
For wide tables, the content can scroll left or right -- which is great, I use wide tables quite often -- but the header and the footer (footnotes area) scrolls along with the table body content. It would be great if the top and bottom elements were fixed in place. What do you think of this idea? Pretty good huh? :)

**Comment by rich-iannone** [comment]
Yes, very good! I see the frustration and this is a great solution. 

I've been thinking a lot lately about the interaction between the header, footer, and table body. It seems like the horizontal relationship should be revisited a bit. I mean, for narrow tables, the header and footer are the same width as the content (it doesn't look too good); there should be width options for each of these three components (e.g., variable for body, 100% for header and footer).

Thanks for your request here. It's a great idea!

**Comment by klarchristian** [comment]
I think as an addition it would be nice if you can collapse/hide column groups (like in Excel). That would help structure various parts of the same table. Is that feasible?
